web/libs/editor/src/mixins/SelectedChoiceMixin.js

```diff
@@ -40,6 +39,12 @@ const SelectedChoiceMixin = types.model().views((self) => ({
       // as they are using alias lookups for choices. For now we will keep it as is since it works for all the
       // other cases currently.
       if (self.findLabel) {
```

Comment:
This `findLabel` is not used in this new code. Better to move new code upper before this check, this will be a better fit.